frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -1005,12 +1024,15 @@ class TableInteractive extends Component {
             }
             return (
               <TableInteractiveRoot
```

Comment:
same here in regards to using data attributes. I think it'll be a lot less polluted in  `classNames` which will keep things relatively readable for anyone looking at the CSS, and will help make tests easier to fix if something happens